src/main/java/org/whispersystems/textsecuregcm/WhisperServerService.java

```diff
@@ -148,6 +151,8 @@ public void run(WhisperServerConfiguration config, Environment environment)
     environment.getObjectMapper().setVisibility(PropertyAccessor.ALL, JsonAutoDetect.Visibility.NONE);
     environment.getObjectMapper().setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
 
+    Optional<ContactDiscoveryConfiguration> cdsConfig = Optional.fromNullable(config.getContactDiscoveryConfiguration());
```

Comment:
Do we ever want this to be optional?

Reply:
do not want this to be optional 
done in ce5c0e04ffab3666ece274b9746efd2dd08d6548